src/components/ConfirmDialog.js

```diff
@@ -0,0 +1,23 @@
+import { h } from 'preact'
+import { useI18n, useSoftkey } from 'hooks'
+
+export const ConfirmDialog = ({ message, onSubmit, close }) => {
+  const i18n = useI18n()
+
+  useSoftkey('ConfirmDialog', {
+    right: i18n.i18n('softkey-close'),
+    onKeyRight: close,
+    center: i18n.i18n('softkey-ok'),
+    onKeyCenter: () => { onSubmit(); close() }
+  }, [])
+
+  return (
+    <div class='confirm-dialog'>
+      <div class='item'>
```

Comment:
Why do we have this hierarchy or `.item > .info > .title` to display one string?

Reply:
good catch, i was copying this from listview, and we don't need item as the class name!

I also make the change to fit longer section name
<img width="179" alt="Capture" src="https://user-images.githubusercontent.com/2560096/71728504-8ce63000-2e3d-11ea-96c0-a340cb806210.PNG">